src/uikit.cs

```diff
@@ -7360,7 +7369,7 @@ interface UIImage : NSSecureCoding
 		[iOS (8,0)]
 		[Export ("imageAsset")]
 		[ThreadSafe]
-		UIImageAsset ImageAsset { get; }
+		UIImageAsset ImageAsset { [NullAllowed] get; }
```

Comment:
Question here, not need to change. Why on the getter and not in the property. In case we add a setter in the future, we won't need to edit anything else.

Reply:
There's no reason really, that's just how I ended up doing it. Presumably it would work just as well on the property.